set.mm

```diff
@@ -471102,6 +471246,54 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+    Transpositions
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    pmtridf1o.a $e |- ( ph -> A e. V ) $.
+    pmtridf1o.x $e |- ( ph -> X e. A ) $.
+    pmtridf1o.y $e |- ( ph -> Y e. A ) $.
+    pmtridf1o.t $e |- T = if ( X = Y , ( _I |` A )
+                                          , ( ( pmTrsp ` A ) ` { X , Y } ) ) $.
+    $( Transpositions of ` X ` and ` Y ` , extended with the identity when they
+       are equal, are bijections.  (Contributed by Thierry Arnoux,
+       1-Jan-2022.) $)
```

Comment:
Do you mean: "the function transposing two possibly equal elements of a set is a bijection of that set" ?

Reply:
See [`~ pmtrfv`](http://us2.metamath.org:88/mpeuni/pmtrfv.html) or [`~ pmtrprfv`](http://us2.metamath.org:88/mpeuni/pmtrprfv.html) : transposition are only defined on actual pairs. If `X = Y`, then ` { X , Y } ~~ 1o ` and we are out of domain.

I wanted to emphasise that I'm completing the domain for `X = Y` (in that case, this is not per se a transposition, and its sign is 0).

Reply:
Ok. Then maybe the phrasing I proposed for the next comment is better. (I don't like the "extended with the identity".)